assets/js/blocks/cart-checkout-shared/view-switcher/utils.ts

```diff
@@ -15,6 +15,21 @@ export const getView = (
 	return views.find( ( view ) => view.view === viewName );
 };
 
+export const selectView = ( clientId: string, viewName: string ) => {
+	const { updateBlockAttributes, selectBlock } =
+		dispatch( 'core/block-editor' );
+	updateBlockAttributes( clientId, {
+		currentView: viewName,
+	} );
+	selectBlock(
+		select( 'core/block-editor' )
+			.getBlock( clientId )
+			?.innerBlocks.find(
+				( block: { name: string } ) => block.name === viewName
+			)?.clientId || clientId
+	);
```

Comment:
This should be optional when happening from sideEffects, maybe a third param to `selectView`?

Because of it, if you select a block, it will not be selected and the parent view is selected, you need to select twice